.github/workflows/build.yaml

```diff
@@ -0,0 +1,66 @@
+name: Build and Push Image for Multi-Arch CI- Additional support for s390x & Power along with arm64 & amd64
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - 'release-[0-9]+\.[0-9]+\.[0-9]+'
+    tags:
+      - 'latest'
+      - 'v*'
+
+
+env:
+  IMAGE_NAME: "odh-model-controller"
+  REPO_NAME: "opendatahub"
+  REGISTRY_NAME: "quay.io"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v3
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_ACCESS_TOKEN }}
```

Comment:
In our other repos, we are using `QUAY_PASSWORD` rather than access token ([example](https://github.com/opendatahub-io/kserve/blob/fb57c15f869ac0bee3b134d206f696a26e35e723/.github/workflows/kserve-controller-docker-publish.yml#L67-L68)).

May you replace `QUAY_ACCESS_TOKEN` with `QUAY_PASSWORD` for consistency?

Reply:
@Nash-123 Please address this change

Reply:
@israel-hdez  I've replaced `QUAY_ACCESS_TOKEN` with `QUAY_PASSWORD` and have updated the PR description to include instructions for creating the secret. Please review and let us know if there are any further changes needed.